diff --git a/node_modules/@runonflux/utxo-lib/src/ecdsa.js b/node_modules/@runonflux/utxo-lib/src/ecdsa.js
index 8841f6e..12655c1 100644
--- a/node_modules/@runonflux/utxo-lib/src/ecdsa.js
+++ b/node_modules/@runonflux/utxo-lib/src/ecdsa.js
@@ -75,7 +75,8 @@ function deterministicGenerateK (hash, x, checkSig) {
 var N_OVER_TWO = secp256k1.n.shiftRight(1)
 
 function sign (hash, d) {
-  typeforce(types.tuple(types.Hash256bit, types.BigInt), arguments)
+  // disable as production code is not working with this line
+  // typeforce(types.tuple(types.Hash256bit, types.BigInt), arguments)
 
   var x = d.toBuffer(32)
   var e = BigInteger.fromBuffer(hash)
diff --git a/node_modules/@runonflux/utxo-lib/src/ecpair.js b/node_modules/@runonflux/utxo-lib/src/ecpair.js
index fea5ae9..26960c7 100644
--- a/node_modules/@runonflux/utxo-lib/src/ecpair.js
+++ b/node_modules/@runonflux/utxo-lib/src/ecpair.js
@@ -31,7 +31,8 @@ function ECPair (d, Q, options) {
 
     this.d = d
   } else {
-    typeforce(types.ECPoint, Q)
+    // disable as production code is not working with this line, todo: fix figure out why. Build disable terser not working. TfTypeError$1: message: "Expected Point, got Point$3"
+    // typeforce(types.ECPoint, Q)
 
     this.__Q = Q
   }
diff --git a/node_modules/@runonflux/utxo-lib/src/ecsignature.js b/node_modules/@runonflux/utxo-lib/src/ecsignature.js
index c74dfe3..aa3dd4c 100644
--- a/node_modules/@runonflux/utxo-lib/src/ecsignature.js
+++ b/node_modules/@runonflux/utxo-lib/src/ecsignature.js
@@ -5,7 +5,7 @@ var types = require('./types')
 var BigInteger = require('bigi')
 
 function ECSignature (r, s) {
-  typeforce(types.tuple(types.BigInt, types.BigInt), arguments)
+  // typeforce(types.tuple(types.BigInt, types.BigInt), arguments)
 
   this.r = r
   this.s = s
diff --git a/node_modules/@runonflux/utxo-lib/src/fastcurve.js b/node_modules/@runonflux/utxo-lib/src/fastcurve.js
index b6e0621..854da47 100644
--- a/node_modules/@runonflux/utxo-lib/src/fastcurve.js
+++ b/node_modules/@runonflux/utxo-lib/src/fastcurve.js
@@ -46,7 +46,7 @@ var publicKeyCreate = function (buffer, compressed) {
  * @return {ECSignature}
  */
 var sign = function (hash, d) {
-  typeforce(types.tuple(types.Buffer256bit, types.BigInt), arguments)
+  // typeforce(types.tuple(types.Buffer256bit, types.BigInt), arguments)
 
   if (!available) {
     return undefined
diff --git a/node_modules/@runonflux/utxo-lib/src/hdnode.js b/node_modules/@runonflux/utxo-lib/src/hdnode.js
index 9ee4ec9..e2e2d8e 100644
--- a/node_modules/@runonflux/utxo-lib/src/hdnode.js
+++ b/node_modules/@runonflux/utxo-lib/src/hdnode.js
@@ -17,7 +17,8 @@ var bs58checkBase = require('bs58check/base')
 var fastcurve = require('./fastcurve')
 
 function HDNode (keyPair, chainCode) {
-  typeforce(types.tuple('ECPair', types.Buffer256bit), arguments)
+  // disable as production code is not working with this line. typeforce has a readme about it.
+  // typeforce(types.tuple('ECPair', types.Buffer256bit), arguments)
 
   if (!keyPair.compressed) throw new TypeError('BIP32 only allows compressed keyPairs')
 
